EIPS/eip-1.md

```diff
@@ -135,113 +134,87 @@ Random J. User
+Many EEPs are written and maintained by developers with write access to the Ethereum codebase. The EEP editors monitor EEP changes, and correct any structure, grammar, spelling, or markup mistakes we see.
 
-The editors don't pass judgment on EIPs. We merely do the administrative & editorial part.
+The editors don't pass judgment on EEPs. We merely do the administrative & editorial part.
 
 ## History
 
-This document was derived heavily from [Bitcoin's BIP-0001] written by Amir Taaki which in turn was derived from [Python's PEP-0001]. In many places text was simply copied and modified. Although the PEP-0001 text was written by Barry Warsaw, Jeremy Hylton, and David Goodger, they are not responsible for its use in the Ethereum Improvement Process, and should not be bothered with technical questions specific to Ethereum or the EIP. Please direct all comments to the EIP editors.
-
-December 7, 2016: EIP 1 has been improved and will be placed as a PR.
-
-February 1, 2016: EIP 1 has added editors, made draft improvements to process, and has merged with Master stream.
-
-March 21, 2018: Minor edits to accommodate the new automatically-generated EIP directory on [eips.ethereum.org](http://eips.ethereum.org/).
-
-May 29, 2018: A last call process was added.
-
-See [the revision history for further details](https://github.com/ethereum/EIPs/commits/master/EIPS/eip-1.md), which is also available by clicking on the History button in the top right of the EIP.
-
-### Bibliography
-
-[EIP5]: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-5.md
-[EIP101]: https://github.com/ethereum/EIPs/issues/28
-[EIP90]: https://github.com/ethereum/EIPs/issues/90
-[EIP86]: https://github.com/ethereum/EIPs/issues/86#issue-145324865
-[devp2p]: https://github.com/ethereum/wiki/wiki/%C3%90%CE%9EVp2p-Wire-Protocol
-[EIP8]: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-8.md
-[Light Ethereum Subprotocol]: https://github.com/ethereum/wiki/wiki/Light-client-protocol
-[whisper]: https://github.com/ethereum/go-ethereum/wiki/Whisper-Overview
-[swarm]: https://github.com/ethereum/go-ethereum/pull/2959
-[API/RPC]: https://github.com/ethereum/wiki/wiki/JSON-RPC
-[EIP59]: https://github.com/ethereum/EIPs/issues/59
-[EIP6]: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-6.md
-[contract ABIs]: https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI
-[interfaces repo]: https://github.com/ethereum/interfaces
-[ERC20]: https://github.com/ethereum/EIPs/issues/20
-[ERC26]: https://github.com/ethereum/EIPs/issues/26
-[ERC137]: https://github.com/ethereum/EIPs/issues/137
-[ERC67]: https://github.com/ethereum/EIPs/issues/67
-[EIP82]: https://github.com/ethereum/EIPs/issues/82
-[EIP75]: https://github.com/ethereum/EIPs/issues/75
-[EIP85]: https://github.com/ethereum/EIPs/issues/85
-[the Ethereum subreddit]: https://www.reddit.com/r/ethereum/
```

Comment:
I would keep those ones and update them:

* `[the EOS subreddit]`: https://www.reddit.com/r/eos/
* `[one of the EOS telegram chat rooms]`: `Does telegram have that`?
* `[pull request]`: https://github.com/eoscanada/EEPs/pulls
* `[the Issues section of this repository]`: https://github.com/eoscanada/EEPs/issues

And of course, maybe changing `eoscanada` to future real owner of the repository